<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
background-color: rgb(106, 106, 106);
font: 63 12pt &quot;Segoe UI Semibold&quot;;
color:white;}</string>
   </property>
   <widget class="QLineEdit" name="email">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>400</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 16pt &quot;Segoe UI Semibold&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>210</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 63 18pt &quot;Segoe UI Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>Hasło</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>200</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 16pt &quot;Segoe UI Semibold&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addAccountButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>680</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 18pt &quot;Segoe UI Semibold&quot;;
border-radius:20px;
background-color: rgb(146, 145, 145);
color:white;</string>
    </property>
    <property name="text">
     <string>Dodaj</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>410</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 63 18pt &quot;Segoe UI Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>80</y>
      <width>411</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(106, 106, 106);

color:white;
font: 63 32pt &quot;Segoe UI Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>Stwórz nowe konto</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>300</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 63 18pt &quot;Segoe UI Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>Powtórz hasło</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="confirm_password">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>290</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 16pt &quot;Segoe UI Semibold&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>450</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 63 18pt &quot;Segoe UI Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>Stanowisko</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>560</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 63 18pt &quot;Segoe UI Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>Czy admin ? </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 63 18pt &quot;Segoe UI Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>Imię</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>210</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 16pt &quot;Segoe UI Semibold&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>300</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 63 18pt &quot;Segoe UI Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>Nazwisko</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="surname">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>290</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 16pt &quot;Segoe UI Semibold&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="error_msg">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>640</y>
      <width>561</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="position">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>450</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 16pt &quot;Segoe UI Semibold&quot;;</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="permisions">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>560</y>
      <width>50</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
